AirCasting/ABConnector/SDCardAirBeamServices.swift

```diff
@@ -104,7 +105,33 @@ class BluetoothSDCardAirBeamServices: SDCardAirBeamServices {
         }
     }
     
-    func finishSync(completion: () -> Void) {
+    func clearSDCard(of peripheral: CBPeripheral, completion: @escaping (Result<Void, Error>) -> Void) {
+        metadataCharacteristicObserver = bluetoothManager.subscribeToCharacteristic(DOWNLOAD_META_DATA_FROM_SD_CARD_CHARACTERISTIC_UUID) { result in
```

Comment:
we are using the same `var` for `downloadData`. It can lead to problems (like if we invoke download and clear one after another the clear will overwrite the old token and it won't get unsubscribed ever). I think it's not intended?

Reply:
Very good catch, thank you!